spec/models/venue_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Venue, type: :model do
+  describe '.save_venue' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.